package.json

```diff
@@ -2,7 +2,7 @@
   "author": "Joshua Holbrook <josh@nodejitsu.com> (http://jesusabdullah.net)",
```

Comment:
yeah, it made me think that you guys sat together and made http-server and ecstatic. I even went to freenode.org#nodejitsu and asked around for you a few times.